permissions/build-time-blame.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "build-time-blame"
+paths:
+- "org/jenkins-ci/build-time-blame"
+developers:
+- "LTegtmeier"
```

Comment:
Your LDAP user name is all lowercase. Not sure whether that makes a difference, but please fix it when you're addressing the groupId issue as well.